lib/tasks/buildThemes.js

```diff
@@ -34,10 +35,8 @@ function getPool(taskUtil) {
 				// There are many stats that could be used, but these ones seem the most
 				// convenient. When all the (available) workers are idle, then it's safe to terminate.
 				const {idleWorkers, totalWorkers} = pool.stats();
```

Comment:
Are those properties updated automatically, or how does the loop work without calling `pool.stats()` again?

Reply:
Yep, sorry, didn't sync with the latest change.
Thanks!

Reply:
I guess the tests didn't fail because one 100ms delay is already enough to clear the queue?